lighty-modules/lighty-gnmi/lighty-gnmi-device-simulator/README.md

```diff
@@ -162,3 +162,30 @@ This example will show how to execute basic operations on lighty.io gNMI device
      ]
    }
    ```
+   
+   ## gNOI
+   Simulator implements the following gNOI gRPCs with implemented logic:
```

Comment:
Simulator implements the following gNOI gRPCs:

Reply:
Done